Makefile

```diff
@@ -423,6 +385,17 @@ ifdef CODE_SIGN_CERTIFICATE_PASSWORD
 endif
 endif
 
+assets/osx/installer.tiff: assets/osx/installer.png assets/osx/installer@2x.png
+	tiffutil -cathidpicheck $^ -out $@
+
+$(BUILD_DIRECTORY)/$(APPLICATION_NAME)-$(APPLICATION_VERSION).dmg: assets/osx/installer.tiff
+	PATH=$(NODE_MODULES_BIN):$$PATH CSC_NAME="$(CODE_SIGN_IDENTITY)" CSC_IDENTITY_AUTO_DISCOVERY=false \
+		build --mac dmg $(ELECTRON_BUILDER_OPTIONS)
+
+$(BUILD_DIRECTORY)/$(APPLICATION_NAME)-$(APPLICATION_VERSION)-mac.zip: assets/osx/installer.tiff
+	PATH=$(NODE_MODULES_BIN):$$PATH CSC_NAME="$(CODE_SIGN_IDENTITY)" CSC_IDENTITY_AUTO_DISCOVERY=false \
+		build --mac zip $(ELECTRON_BUILDER_OPTIONS)
```

Comment:
The ZIP is required to auto-updating purposes. We've been looking to do that since a long time ago, which is the reason why we're publishing it. Moving to `electron-builder` should make this feature a bit easier to implement, and then ZIPs will be very widely used.